__tests__/app.test.js

```diff
@@ -0,0 +1,25 @@
+const request = require("supertest");
+const testData = require("../db/data/test-data/index.js");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed");
+
+const app = require("../app");
+
+beforeEach(() => seed(testData));
+afterAll(() => connection.end());
+
+describe("/api/topics", () => {
+  describe("GET", () => {
+    test("200 OK: should return an array of all the topics", () => {
+      return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+          body.topics.forEach((topic) => {
```

Comment:
What if the array of topics came back empty? The assertions on the line below would never be reached and would therefore never fail. How can we check the array isn't empty first?